apm-lambda-extension/apmproxy/receiver_test.go

```diff
@@ -48,13 +49,17 @@ func TestInfoProxy(t *testing.T) {
 	}))
 	defer apmServer.Close()
 
+	l, err := logger.New()
+	require.NoError(t, err)
```

Comment:
Instead of repeating this in all the tests, can we use [`zaptest.NewLogger`](https://pkg.go.dev/go.uber.org/zap@v1.21.0/zaptest#NewLogger)? That could be used inline in the `WithLogger` call to keep the tests a little bit neater.

Reply:
TIL about `zaptest`! It looks much cleaner now 👍